src/components/Paging/Paging.tsx

```diff
@@ -0,0 +1,74 @@
+import * as React from 'react';
+import { Button } from '@rmwc/button';
+import styles from './Paging.module.css';
+import { useState } from 'react';
+
+interface PagingProps {
+  pages: number;
+  pageToStartWith: number;
+  onPageChange: (newPage: number) => void;
+  pagesToDisplayFrom?: number; // TODO?
```

Comment:
should we implement this?
(so that you can only display a subset of pages like 1-10 (if there are 30 pages for example. Maybe that would be better with a bool flag and an integer number prop though.)

Reply:
I don't get it

Reply:
I'll explain tomorrow. This would only be kinda relevant as soon as you have more than 10 pages probably.

Reply:
Probably not really worth it (at least right now)

Reply:
![image](https://user-images.githubusercontent.com/16801528/57253053-0d098b00-704e-11e9-966c-1c2a3a1b2c5d.jpeg)
I mean something like this.